frontend/WikiContrib-Frontend/src/result.js

```diff
@@ -709,77 +726,107 @@ class QueryResult extends React.Component {
+                                      <div className="chart_message">
+                                      <h2>{this.state.meta.full_names.phab_full_name}</h2>
+                                      </div>
+                                    )}
+                                  </span>
+                                </Card>
+                              )}
+                            </div>
+                            <div className="graph">
+                              {this.state.loading ? (
+                                <Card className="graph_load">
+                                  <Card.Content>
+                                    <Placeholder fluid className="image_load">
+                                      <Placeholder.Line />
+                                    </Placeholder>
+                                  </Card.Content>
+                                </Card>
+                              ) : (
+                                <Card className="chart_container">
+                                  <span style={{ textAlign: 'center' }}>
+                                    <Header className="chart"> GERRIT </Header>
+                                    {this.state.meta.full_names.gerrit_full_name !== "no username provided" &&
+                                     this.state.meta.full_names.gerrit_full_name !== "username does not exist" ? (
+                                      <Line
+                                        ref="chart"
+                                        data={this.getGraphData('gerrit')}
+                                        options={chartOptions}
+                                      />
+                                    ) : (
+                                      <div className="chart_message">
+                                      <h2>{this.state.meta.full_names.gerrit_full_name}</h2>
+                                      </div>
+                                    )}
+                                  </span>
+                                </Card>
+                              )}
+                            </div>
+                            <div className="graph">
+                              {this.state.loading ? (
+                                <Card className="graph_load">
+                                  <Card.Content>
+                                    <Placeholder fluid className="image_load">
+                                      <Placeholder.Line />
+                                    </Placeholder>
+                                  </Card.Content>
+                                </Card>
+                              ) : (
+                                <Card className="chart_container">
+                                  <span style={{ textAlign: 'center' }}>
+                                    <Header className="chart"> GITHUB </Header>
+                                    {this.state.meta.full_names.github_full_name !== "no username provided" &&
```

Comment:
Change both the error messages here as well. If possible, can you change the error messages for Phabricator and Gerrit as well in this PR for consistency. If not file a PR, and it can be handled separately.

Reply:
Will make the changes in this PR